src/index.js

```diff
@@ -24,7 +24,7 @@ module.exports = {
       plugins: ['skyscanner-dates'],
       rules: {
         'skyscanner-dates/no-saddlebag-date': 'warn',
```

Comment:
```suggestion
        'skyscanner-dates/no-saddlebag-date': 'error',
```
`saddlebag-date` will be deprecated, it needs to be set `error`.